responsibleai/responsibleai/_interfaces.py

```diff
@@ -20,6 +20,8 @@ class Dataset:
     class_names: List[str]
     categorical_features: List[str]
     target_column: str
+    is_large_data_scenario: bool
```

Comment:
should this be optional? asking because if it will break existing flow?

Reply:
going forward SDK will always add this so it might be ok if it is not optional. It shouldn't break any existing flow as this is not being used anywhere. 

The UI treats this as Optional property anyways in the PR:- https://github.com/microsoft/responsible-ai-toolbox/pull/1804.